### PR TITLE
WCAG 2.1 support

### PR DESCRIPTION
**Pull request by dmundra**
Summary of changes:
* Updated licenses and put back report date logic for templates.
* Created WCAG 2.1 catalog and build the VPAT catalogs for WCAG and WCAG2.1+508.
* Added a field to track which catalog will be used by the editor.
* Updated the create catalog function and tests for the librarian function to build the 3 different catalogs.

@mgifford please review the 2.1 catalogs for anything I missed to capture from https://www.w3.org/TR/WCAG21/

Progress on https://github.com/GSA/openacr/issues/270

### CODE REVIEW

File: src/createCatalog.ts [hunk 1]
```diff
@@ -29,26 +31,26 @@ function getComponents(components: any): any {
...
   }
 }
 
 function getStandards(first: any, second: any): any {
-  if (validateCatalogDataFiles(first) && validateCatalogDataFiles(second)) {
-    return first.standard.concat(second.standard);
-  }
-}
-
-function getTitle(data: any): any {
```
Comment (mgifford) [inline src/createCatalog.ts:43]: Why did getTitle & getLang get nixed here?

Comment (dmundra) [reply src/createCatalog.ts:43]: I was lazily building it from the data files, instead I now pass it when calling `createCatalog` from the librarian file.

File: templates/openacr-html-0.1.0.handlebars [hunk 2]
```diff
@@ -29,6 +29,8 @@
       </h2>
       {{#if report_date}}
         {{report_date}}
+      {{else}}
+        {{ now }}
```
Comment (mgifford) [inline templates/openacr-html-0.1.0.handlebars:33]: Nice to have this fallback.

**Review by mgifford — APPROVED** [review]
I scanned through the files but couldn't find anything out of line.

**Pull request merged**